### DISCUSSION TITLE
Is it possible to do Multi Tenancy in SecurityOnion?

### DISCUSSION THREAD

**Discussion by hkcs1147** [Security-Onion-Solutions/securityonion#12573] [2.4]
### Version

2.4.50

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

other (please provide detail below)

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

8

### RAM

16

### Storage for /

250

### Storage for /nsm

250

### Network Traffic Collection

tap

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hello, I wanted to ask is it possible to do Multi-Tenancy in Security Onion? I am having few clients and I wanted each of those clients to only view specific data on their Kibana Dashboard? 

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by cm-ops** [comment]
There is currently no support for multi-tenancy.

**Reply by hkcs1147** [reply]
Thank you so much for the answer.
